le-taxi-api-tests/src/taxiPositionSnapShots/invalidTaxiPositionSnapShots.apiTest.ts

```diff
@@ -17,6 +17,20 @@ export async function invalidTaxiPositionSnapShotsTests(): Promise<void> {
   testCreateTaxiPositionSnapShotsUserAccessInvalid(UserRole.Stats);
   testCreateTaxiPositionSnapShotsUserAccessInvalid(UserRole.Prefecture);
 
+  it('should return error when operator sends a taxi that doesnt exist.', async () => {
+    const [dtoTaxiPositionSnapShot, apiKey] = await getValidTaxiPositionSnapshotDtoAndApikey(UserRole.Operator);
```

Comment:
Only 1 position snapshot here. Would be interesting to have more than one here. The error message seems to suggest that there can be 1 bad  in a bunch of many position snapshot